apps/src/templates/sectionAssessments/sectionAssessmentsRedux.js

```diff
@@ -466,6 +466,33 @@ export const getMultipleChoiceSectionSummary = (state) => {
   return results;
 };
 
+/**
+ * Selector function that takes in the state and a boolean indicating
+ * if the current assessment is a survey.
+ * Returns an integer total count of the number of students who have submitted
+ * the current assessment.
```

Comment:
I'd prefer [JSDoc `@returns`](http://usejsdoc.org/tags-returns.html) for consistency.

```js
/**
 * @returns {number} count of the number of students who have submitted
 *   the current assessment.
 */
```